ja-odur/puzzled
# #167557229 add navbar to sudoku page

**What does this PR do?**
* Add navigation bar to the sudoku page
* Clear browser errors
* Remove the 1x1 sudoku grid

**Description of Task to be completed?**
* Add navigation bar to the sudoku page
* Clear browser errors
* Remove the 1x1 sudoku grid

**How should this be manually tested?**
* Pull and checkout to this branch locally by running `git fetch origin ft-sudoku-nav-footer-167557229 && ft-sudoku-nav-footer-167557229`
* Start  the virtual environment `pipenv shell`
* install python requirements `pipenv install`
* install node requirements `npm install`
* migrate the database `python manage.py migrate`
* run builds `npm run build`
* start the django server `python manage.py runserver`
* Hit the endpoint `http://127.0.0.1:8000/sudoku/`, the page should contain a navbar.

**Any background context you want to provide?**
* N/A

**What are the relevant pivotal tracker stories?**
* [167557229](https://www.pivotaltracker.com/story/show/167557229)
